website/docs/reference/warehouse-profiles/trino-profile.md

```diff
@@ -42,27 +44,30 @@ trino:
   outputs:
     dev:
       type: trino
-      method: none  # optional, one of {none | ldap | kerberos}
+      method: none  # optional, one of {none | ldap | kerberos | oauth | jwt | certificate}
       user: [user]
       password: [password]  # required if method is ldap or kerberos
       database: [database name]
       host: [hostname]
       port: [port number]
       schema: [your dbt schema]
       threads: [1 or more]
+      retries: [1 or more]
       http_scheme: [http or https]
```

Comment:
might be worth calling out what the default values for these fields are

Reply:
Good point, in which form this could be presented? Do you have some suggestions from other adapters?

Reply:
@hovaesco check out the example in [Snowflake's profile page](https://docs.getdbt.com/reference/warehouse-profiles/snowflake-profile#all-configurations)?

Reply:
Done